src/Logger.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Component;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\GelfHandler;
+use Monolog\Handler\HandlerInterface;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger as MonologLogger;
+use function array_filter;
+use function array_values;
+
+class Logger extends MonologLogger
+{
+    /**
+     * @param HandlerInterface[] $handlers
+     */
+    public function __construct(
+        array $handlers = []
+    ) {
+        parent::__construct('php-component');
+
+        // only keep valid handlers
+        $handlers = array_filter(
+            array_values($handlers),
+            function ($handler) {
+                return $handler instanceof HandlerInterface;
+            }
+        );
+
+        // no handlers
+        if (count($handlers) === 0) {
+            $handlers = [
+                new StreamHandler('php://stderr'),
+                new StreamHandler('php://stdout'),
+            ];
+        }
+
+        // gelf log handler
+        if (count($handlers) === 1 && !($handlers[0] instanceof GelfHandler)) {
+            throw new \Exception(
+                'If only one handler is provided, it needs to be GelfHandler'
+            );
+        }
+
+        // two handlers (default)
+        if (count($handlers) === 2) {
```

Comment:
aha, no ja jsem to myslel tak, ze pokud si tam posles vlastni handlery, tak tam mas ty - tohle se me zda dost nepruhledny

Reply:
Jo, už jsem to upravil.

Reply:
Jediná nevýhoda je, že tím pádem už není co testovat. Protože když si nastavím v testu testHandlery, tak testuju jen, že jsem nastavil správně test a ne default nastavení :/

Reply:
aha, a neslo by to udelat tak jak jsi to mel predtim s tema streamama?

Reply:
No můžu si tam dát ty test streamy a nastavit jim level a bubble, ale tím testuju, že jsem správně nastavil ty TestStreamy. Netestuju, že když tam pošlu prázdné pole, tak to bude správně.

Reply:
jo to chapu, myslel jsem nejakej rewind stdout/stderr, ale to asi nejde no